### PR TITLE
Added url strings for homepage and documentation

### LINKED ISSUE #603: Cannot install with pip (data.homepage must be uri)
When trying to install `qibocal` with `pip` an error is raised, stating that `homepage` and `documentation` in `pyproject.toml` must be URL strings.

### PR DESCRIPTION
**Pull request by GabrielePalazzo**
Closes #603.

Let me know if it is a reasonable solution or if you want to discard it.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -5,9 +5,9 @@ description = "https://qibo.science/"
 authors = ["andrea-pasquale <andreapasquale97@gmail.com>"]
 license = "Apache License 2.0"
 readme = "README.md"
-homepage = ""
+homepage = "https://github.com/qiboteam/qibocal/"
```
Comment (scarrazza) [inline pyproject.toml:8]:
```suggestion
homepage = "https://qibo.science/"
```

File: pyproject.toml [hunk 2]
```diff
@@ -5,9 +5,9 @@ description = "https://qibo.science/"
...
 repository = "https://github.com/qiboteam/qibocal/"
-documentation = ""
+documentation = "https://qibo.science/"
```
Comment (scarrazza) [inline pyproject.toml:10]:
```suggestion
documentation = "https://qibo.science/qibocal/stable/"
```

**Comment by scarrazza** [comment]
Thanks, could you please also replace the description with " Qibo's quantum calibration, characterization and validation module."?

**Review by andrea-pasquale — APPROVED** [review]

**Review by scarrazza — APPROVED** [review]

**Comment by GabrielePalazzo** [comment]
@scarrazza @andrea-pasquale please have a final look, since something weird happened with git (it reverted some previous commits). Now it should all be ok.

**Comment by andrea-pasquale** [comment]
The final version looks good to me. I've just relaunched the CI. Feel free to merge as soon as tests are passing.

**Pull request merged**